tests/framework/e2e/etcdctl.go

```diff
@@ -435,6 +459,20 @@ func (ctl *EtcdctlV3) AlarmDisarm(_ *clientv3.AlarmMember) (*clientv3.AlarmRespo
 	return &resp, err
 }
 
+func (ctl *EtcdctlV3) AuthEnable() (*clientv3.AuthEnableResponse, error) {
+	return &clientv3.AuthEnableResponse{}, SpawnWithExpect(ctl.cmdArgs("auth", "enable"), "Authentication Enabled")
```

Comment:
It doesn't make sense to return a empty `&clientv3.AuthEnableResponse{}` response. You have two choice:
1. Only return an error for the method;
2. Populate the `&clientv3.AuthEnableResponse{}` using `spawnJsonCmd`

Reply:
Good call. Will go with option 1